content/docs/concepts/security/index.md

```diff
@@ -223,86 +223,89 @@ origins:
 
 Defines what is the principal from the authentication. By default, this will be the peer's principal (and if peer authentication is not applied, it will be left unset). Policy writers can choose to overwrite it with USE_ORIGIN. In future, we will also support *conditional-binding* (e.g USE_PEER when peer is X, otherwise USE_ORIGIN)
 
-## Role-Based Access Control (RBAC)
+## Istio Authorization
```

Comment:
Auth -> auth

Reply:
This is the start of Istio authorization feature. Just like "Mutual TLS Authentication" and "Authentication Policy" sections, I would like to keep it upper case.

Reply:
I think we should fix the other header to comply with our pattern. Authorization is not a proper noun and shouldn't be capitalized, either in titles or in the main text. "Authentication Policy" is similar, and should just be "Authenthentication policy".

See https://preliminary.istio.io/about/contribute/style-guide/#only-capitalize-the-first-letter-of-headings

Reply:
Fixed the titles of other sections in the file.